src/events/event_handler.ts

```diff
@@ -127,6 +128,59 @@ export class eventHandler {
 		this.handleChanges(filePath, currentText);
 	}
 
+	handleTabChangeEvent = () => {
+		/*
+		The flow for handler will be as follows:
+			- For the current open repoPath, get the repo_id and from config File
+			- Get Current Open Tabs TabGroups.ALL
+			- For each tab:
+				* Get the Relative Path of each file using repoPath
+				* Get the fileID using relative_path found in above step
+		*/
+		// Record timestamp
+		const created_at = new Date().getTime();
+		console.log("created_at: ", created_at);
+		// For the current open repoPath, get the repo_id and from config File
+		const configUtils = new ConfigUtils();
+		const repoId = configUtils.getRepoIdByPath(this.repoPath);
+		console.log(`Repo ID: ${repoId}`)
+		const configJSON = configUtils.config;
+
+		// Get list of current tabs
+		const open_tabs = vscode.window.tabGroups.all;
+		let tabs: any[] = [];
```

Comment:
why not `const`?

Reply:
No specific reason. I'll update it to `const`